docs/contributing.md

```diff
@@ -43,16 +42,16 @@ preferred.
 
 All changes should be in the Flatworm Markdown Dialect.
 
-_heading: Building the Documentation
+### Building the Documentation
 
 Currently, the documentation is built using an experimental v2 of the
 Flatworm documentation system, a system originally specifically made
 to maintain the quais documentation.
 
-The new ``tsdocs`` branch has the ability to parse ``jsdocs`` from
+The new `tsdocs` branch has the ability to parse `jsdocs` from
 from TypeScript source files to create an API reference.
 
-_code: Building with the v2 Flatworm @lang<shell>
+```bash title=Building with the v2 Flatworm @lang<shell>
```

Comment:
Delete refernces to flatworm

Reply:
Addressed in [f799338](https://github.com/dominant-strategies/quais-6.js/pull/127/commits/f79933813d77192842ed793ec55a5b256e76bac4)